Resources/views/process/_process_global_actions_header.html.twig

```diff
@@ -1,32 +1,37 @@
 <div class="row">
     <div class="col-12">
 
+        {# @todo Styling needed. When we are removing items the sidebar looks out of place #}
```

Comment:
Do we have to address this issue now or can we live with it for now?

Reply:
We actually already addressed it by adding the .side-bar-right class to the core.scss and applying it to the sidebar. The comment didn't get removed in that process, but it's removed now.